bin/exec.go

```diff
@@ -146,7 +146,7 @@ func main() {
 		experiment.JobName = engineDetails.Experiments[i] + "-" + randomString
 
 		log.Infof("JobName for this Experiment : %v", experiment.JobName)
-
+		analytics.TriggerAnalytics(engineDetails.Experiments[i])
```

Comment:
Can you add check if clientUUID is non empty then it will run the analytics also move this in the starting of for loop with porper comments